core/services/fluxmonitor/flux_monitor_test.go

```diff
@@ -1463,6 +1463,54 @@ func TestFluxMonitor_PollingDeviationChecker_HandlesNilLogs(t *testing.T) {
 	})
 }
 
+func TestFluxMonitor_IdleTimer(t *testing.T) {
```

Comment:
Can we have a test on the v2 fix also?

Reply:
there is one - thats what https://github.com/smartcontractkit/chainlink/pull/4397/files#diff-79e16dc4b7fdbc22de88a5d4da5a454d4fb0adb9e4e7c0ac73d669b877608e51R115 is